app/controllers/patients_controller.rb

```diff
@@ -155,6 +162,44 @@ def update
     # If we failed to find a subject given the id, redirect to index
     redirect_to(root_url) && return if patient.nil?
 
+    # Propogate desired fields to household except jurisdiction_id
+    propogated_fields = params[:propogated_fields]
```

Comment:
so I wanted to do something along the lines of:
`propogated_fields = params.permit(:propogated_fields).permit(*group_member_subset)`
but that was not passing that field to the controller. I also tried:
`propogated_fields = params.require(:propogated_fields).permit(*group_member_subset)`
but rails would complain when :propogated fields is an empty Hash

so I'm wondering if there's a better way to do this or if I'm just doing it wrong?

Reply:
propogated -> propagated